WordPattern/Patterns.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace WordPattern
+{
+    public class Patterns
+    {
+        public bool FindPattern(string pattern, string patternToCheck)
+        {
+            Dictionary<char, string> dictonary = new Dictionary<char, string>();
+            HashSet<string> set = new HashSet<string>();
```

Comment:
Bad naming, dictionary and set is misleading name, Please rename to sth more meaningful